test/mocha.opts

```diff
@@ -0,0 +1,3 @@
+--recursive
+--timeout 20000
+--ui tdd
```

Comment:
[`\n`][1], please. :)


[1]: https://github.com/editorconfig/editorconfig/wiki/Newline-at-End-of-File-Support